pytorch_lightning/trainer/trainer_io.py

```diff
@@ -93,6 +96,18 @@ def restore_state_if_checkpoint_exists(self, model):
 
         return did_restore
 
+    def restore_state_from_checkpoint(self, checkpoint_path):
+        did_restore = False
+
+        checkpoint_path = Path(checkpoint_path)
+        if not checkpoint_path.exists():
+            return did_restore
```

Comment:
You can simply return True/False no need for extra variable

Reply:
Okay. I'll change the code accordingly.

Reply:
I simplified the code and this function no longer exists